GREEDY/Controllers/UsersController.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System.Net.Http;
+using System.Web.Http;
+using System.Web.Http.Cors;
+using GREEDY.Services;
+using Newtonsoft.Json;
+using GREEDY.Models;
+using GREEDY.Extensions;
+using System;
+using GREEDY.DataManagers;
+using System.Threading.Tasks;
+
+namespace GREEDY.Controllers
+{
+    [EnableCors(origins: "*", headers: "*", methods: "*")]
+    public class LoginController : ApiController
+    {
+        private IAuthService _authService;
+        private ISessionManager _sessionManager;
+        public LoginController(IAuthService authService, ISessionManager sessionManager)
+        {
+            _authService = authService;
+            _sessionManager = sessionManager;
+        }
+        public async Task<HttpResponseMessage> Put()
+        {
+            HttpContent requestContent = Request.Content;
+            string jsonContent = await requestContent.ReadAsStringAsync();
+            LoginCredentials credentials = JsonConvert.DeserializeObject<LoginCredentials>(jsonContent);
+            if (credentials.Username.Length < 5 || credentials.Password.Length < 5
```

Comment:
still magic numbers and not really convenient to change it fast if we would like to.